### PR TITLE
Metadata S3 bucket creation

### PR DESCRIPTION
**Pull request by fwelschen**
In this PR we are adding s3 bucket creation feature to store related metadata for the VPC.
This feature include:
  - enable and disable s3 bucket creation variable
  - s3 bucket name output
  - variable description

### CODE REVIEW

**Comment by fwelschen** [comment]
[Terraform / Kops :: K8s Cluster en Staging](https://trello.com/c/YlyoG7mP/449-terraform-kops-k8s-cluster-en-staging)

File: outputs.tf [hunk 1]
```diff
@@ -158,3 +158,9 @@ output "enable_remote_dns_resolution" {
   description = "True if you have dns resolution between your VPCs"
   value       = "${var.enable_remote_dns_resolution}"
 }
+
+output "vpc_bucket_name" {
+  #count = "${var.enable_s3_bucket ? 1 : 0}"
```
Comment (idontdomath) [inline outputs.tf]: remove.

File: main.tf [hunk 2]
```diff
@@ -329,3 +329,17 @@ resource "aws_vpc_peering_connection" "this" {
 
   tags = "${merge(var.tags, var.vpc_tags, map("Name", format("%s-to-%s", var.name, element(var.vpc_peering_with_name, count.index))))}"
 }
+
+################################
+# Bucket S3 ## We use it for k8s
```
Comment (idontdomath) [inline main.tf]: maybe instead of commenting this as it's specifically for k8s we should mention that it's used for metadata on implementations.

**Review by idontdomath — CHANGES_REQUESTED** [review]
Maybe we should find a more specific name of the variable in order to explain why we are using this, or why we are creating it.

**Review by idontdomath — APPROVED** [review]
🚢 it!

**Pull request merged**